### DISCUSSION TITLE
[Improvement]: Metrics improvements

### DISCUSSION THREAD

**Discussion by peaklabs-dev** [coollabsio/coolify#3155] [Improvement Requests]
### Description
The new metrics feature is amazing.

### Minimal Reproduction (if possible, example repository)

A few small suggestions for the metrics feature:

- [ ] Add a core count above the CPU graph so we know there are 4 cores total and 40% (percentage from the graph) is normally being used (I always forget how many cores I have).
- [ ] Add a ram amount above the ram graph -> so we can see that 60% (graph) of 4Gb of ram is being used normally (text above the graph)
- [ ] Add core count to the containers -> this core count should take into account when we have limited the resources of a container
- [ ] Add a disk usage graph to containers
- [ ] Add a disk usage graph to the server
- [ ] Add a disk sizes above the disk graph -> so we can see that 60% (graph) of 40Gb of disk is being used normally (text above the graph)
- [ ] Add the current ram/ the ram usage as a GB number usage to the ram amount like this 4Gb/2.2Gb (or similar) -> so we can see the total ram and the current/ normally used ram and do not have to start calculating what 26% normal ram usage of our 4GB actually is --> to containers and servers --> ram amount on containers should take into account limits we habe set
- [ ]  Add the current disk usage as a GB number usage to the disk amount like this 40Gb/22Gb (or similar) -> so we can see the total disk usage and the current/ normally used disk usage and do not have to start calculating what 26% of a 40GB disk is.
- [ ] Maybe this is possible, maybe not, but add a network diagram to the server and count how many GB/TB of traffic go through all containers -> so we can see our used network bandwidth (as network bandwidth cost per GB).
- [ ] Add exception notifications -> so we can set a threshold and as soon as CPU usage or RAM usage/ network bandwith... spike over that threshold we get notified

### Version

v4.0.0-beta.306

**Comment by DeVoresyah** [comment]
is this request feature or metrics already in coolify? because I couldn't find any metrics on coolify

**Comment by peaklabs-dev** [comment]
@DeVoresyah Not in the release version, but in the upcoming release 298.

**Comment by RayBB** [comment]
Metrics are released as of today. Must be enabled in server settings.

**Comment by peaklabs-dev** [comment]
Example of how metrics for traffic per site/ entire server could look like:
![image](https://github.com/coollabsio/coolify/assets/122374094/6cb5d893-a2e5-40dc-b43b-641e84efdcb4)

**Comment by nnti3n** [comment]
Would like some more metrics like disk storage, disk io, network io
